src/components/Invitations/InviteProvider.tsx

```diff
@@ -1,144 +1,105 @@
-import React, { createContext, useCallback, useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import { Alert } from 'react-native';
+import { create } from 'zustand';
 import { t } from '../../../lib/i18n';
 import { inviteNotifier } from './InviteNotifier';
 import { useAuth } from '../../hooks/useAuth';
 import { useRestCache, useRestMutation } from '../../hooks/rest-api';
+import { ActivityIndicatorView } from '../ActivityIndicatorView';
+import { useQueryClient } from '@tanstack/react-query';
 
-type InviteParams = {
-  inviteId?: string;
+export type PendingInvite = {
+  inviteId: string;
   evc?: string;
 };
 
-type InviteState = {
-  inviteParams: InviteParams;
-  clearPendingInvite: () => Promise<void>;
-  lastAcceptedId: string;
+const store = create<PendingInvite | undefined>(() => undefined);
```

Comment:
Should we call this `usePendingInviteStore`? That seems to be the convention that zustand uses in its readme.

Reply:
@sternetj I like it. I'll follow-up with that improved naming.